app/models/installed_technology.rb

```diff
@@ -71,7 +71,8 @@ def technology
   def behavior_with_curve(curve_type = nil)
     behavior = self.behavior.presence || technology.behavior
 
-    return behavior if curve_type.nil? || curve_type == 'default'.freeze
+    return "default" if is_edsn_profile?
```

Comment:
I'm not sure I understand why an EDSN profile means `default` behavior (btw, the correct name for the "featureless" behavior is `generic`). Shouldn't it be `default` for the inflexible component and `optional` (saving) for the flexible? If so, this would be accomplished by creating two new TechnologyComponentBehavior records:

```ruby
tech = Technology.find_by_key('base_load_edsn')

tech.component_behaviors.create!(curve_type: 'flex',   behavior: 'optional')
tech.component_behaviors.create!(curve_type: 'inflex', behavior: 'generic')
```